### ISSUE TITLE
Error after parsing less with postcss-less

### ISSUE THREAD

**Issue by zry656565** [shellscape/postcss-less#53]
Repro demo: https://github.com/zry656565/postcss-less-issue-repro

``` bash
# install dependencies
npm install
# run gulp task
gulp
```

**Error Message:**

```
[01:00:46] Starting 'default'...
[01:00:46] 'default' errored after 218 ms
[01:00:46] Error in plugin 'gulp-less'
Message:
    Unrecognised input in file /home/vagrant/postcss-issue-repro/demo.less line no. 8
Details:
    type: Parse
    filename: /home/vagrant/postcss-issue-repro/demo.less
    index: 181
    line: 8
    callLine: NaN
    callExtract: undefined
    column: 13
    extract:   .heading-font,  font-weight: 700;,}
    lineNumber: 8
    fileName: /home/vagrant/postcss-issue-repro/demo.less
[01:00:46] 'default' errored after 225 ms
[01:00:46] Error: task completion callback called too many times
    at finish (/home/vagrant/postcss-issue-repro/node_modules/orchestrator/lib/runTask.js:15:10)
    at /home/vagrant/postcss-issue-repro/node_modules/orchestrator/lib/runTask.js:52:4
    at f (/home/vagrant/postcss-issue-repro/node_modules/once/once.js:17:25)
    at DestroyableTransform.onend (/home/vagrant/postcss-issue-repro/node_modules/end-of-stream/index.js:31:18)
    at emitNone (events.js:85:20)
    at DestroyableTransform.emit (events.js:179:7)
    at /home/vagrant/postcss-issue-repro/node_modules/vinyl-fs/node_modules/readable-stream/lib/_stream_readable.js:965:16
    at nextTickCallbackWith0Args (node.js:453:9)
    at process._tickCallback (node.js:382:13)

```

**Comment by shellscape** [comment]
@zry656565 would you be able to provide the full LESS file that this was erroring on?

**Comment by zry656565** [comment]
I think the problem has been solved in 0.16.1. Thanks!
